### PR TITLE
Add pause token command

### LINKED ISSUE #83: Add prop-based test command for pausing the token
We have to think about the logic on pausing the token a bit more though. We are already pausing the token to prevent transfers while the Crowdsale is ongoing. However, we have to allow minting during the Crowdsale, but at the same time there should be a way to pause the minting too.

### PR DESCRIPTION
**Pull request by nberger**
Fixes #83

### CODE REVIEW

File: test/CrowdsaleGenTest.js [hunk 1]
```diff
@@ -106,7 +111,7 @@ contract('LifCrowdsale Property-based test', function(accounts) {
       (nextBlock > endBlock2) ||
       (state.crowdsalePaused) ||
       (state.crowdsaleFinalized) ||
-      (weiCost == 0);
+      (command.eth == 0);
```
Comment (AugustoL) [inline test/CrowdsaleGenTest.js]: How is this change related to the pause token command?

Comment (nberger) [inline test/CrowdsaleGenTest.js]: not really very related :)
Do you think I should do a separate commit, or even a separate PR?

Comment (AugustoL) [inline test/CrowdsaleGenTest.js]: A separate commit will be enough :)

Comment (nberger) [inline test/CrowdsaleGenTest.js]: @AugustoL ok, better now? :)
